necsim/impls/std/src/event_log/recorder.rs

```diff
@@ -129,6 +129,32 @@ impl EventLogRecorder {
         Ok(self)
     }
 
+    /// # Errors
+    ///
+    /// Fails to construct iff `path` is not a writable directory.
+    pub fn clone_move(&self, path: PathBuf) -> Result<Self> {
```

Comment:
Find a better name, maybe `open_sibling_in`

Reply:
Also, we cannot just clone into a sibling since the parent will remove the directory on being dropped